llama_stack/core/server/server.py

```diff
@@ -238,7 +237,7 @@ async def route_handler(request: Request, **kwargs):
                         result.url = route
                     return result
             except Exception as e:
-                if logger.isEnabledFor(logging.DEBUG):
```

Comment:
I think this is checking the value `logging.DEBUG`, this might be an incorrect change? I could be wrong

Reply:
so the PR removes `import logging` 

please correct me if i am wrong :) 

cc @leseb @ashwinb